lib/wpc_proto/internal_modules/proto_config.c

```diff
@@ -728,56 +728,84 @@ app_proto_res_e Proto_config_handle_set_config(wp_SetConfigReq *req,
         }
     }
 
-    if ( cfg->has_network_address &&
+    if (cfg->has_network_address &&
         (cfg->network_address != m_sink_config.network_address))
     {
-        res = WPC_set_network_address(cfg->network_address);
-        if (res != APP_RES_OK)
+        if (cfg->network_address >= (1ULL << (sizeof(net_addr_t) * 8)) )
```

Comment:
In reality network address is a 3 bytes value (https://github.com/wirepas/wm-sdk-subghz/blob/rel_1.0.0_SubGhz/libraries/dualmcu/api/DualMcuAPI.md#cnetworkaddress).

Even if we store it in a uint32_t.

So testing it based on variable size is not restrictive enough